pkg/cloudprovider/vsphere/loadbalancer/state.go

```diff
@@ -0,0 +1,457 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package loadbalancer
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog"
+
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+
+	"k8s.io/cloud-provider-vsphere/pkg/cloudprovider/vsphere/loadbalancer/config"
+)
+
+var verboseLevel = klog.Level(2)
```

Comment:
We shouldn't need to do this. The cloud-controller-manager is already setup to accept the verbose level via a flag.

Reply:
This only sets the verbose level used for the logging in the state methods. It does not set the global verbose level.

Reply:
Ah I see, thanks for clarifying